lib/jdomodels/src/main/resources/sql/GetEntityAccessRestrictions.sql

```diff
@@ -24,14 +24,23 @@ WITH
 		SELECT EAR.*, if(AA.STATE = 'APPROVED', TRUE, FALSE) AS APPROVED FROM EAR
 			LEFT JOIN ACCESS_APPROVAL AA
 				ON (EAR.REQUIREMENT_ID = AA.REQUIREMENT_ID AND AA.ACCESSOR_ID = :userId AND AA.STATE = 'APPROVED')
-	 )
+	 ),
+    ACC AS(
+        SELECT EAR.REQUIREMENT_ID AS REQUIREMENT_ID, IF(RAT.STRING_ELE ='EXEMPTION_ELIGIBLE', TRUE, FALSE) AS IS_EXAMPTION_ELIGIBLE FROM EAR
+            LEFT JOIN ACL ON (EAR.REQUIREMENT_ID= ACL.OWNER_ID)
```

Comment:
LEFT JOIN ACL ON (EAR.REQUIREMENT_ID= ACL.OWNER_ID AND ACL.OWNER_TYPE = 'ACCESS_REQUIREMENT')